app/src/main/java/com/bignerdranch/android/geoquiz/MainActivity.kt

```diff
@@ -12,12 +9,8 @@ import android.view.View
 import android.widget.Button
 import android.widget.TextView
 import android.widget.Toast
-import androidx.lifecycle.ViewModelProvider
 import androidx.lifecycle.ViewModelProviders
-
-private const val TAG = "MainActivity"
-private const val KEY_INDEX = "index"
-private const val REQUEST_CODE_CHEAT = 0
+import com.bignerdranch.android.geoquiz.CheatActivity.Companion.EXTRA_ANSWER_SHOWN
```

Comment:
You're importing it here, but are you using it anywhere in this class? If not, then it's better to remove this import